thorlcr/activities/firstn/thfirstnslave.cpp

```diff
@@ -215,7 +215,10 @@ class CFirstNSlaveGlobal : public CFirstNSlaveBase, implements ILookAheadStopNot
     rowcount_t maxres = RCUNBOUND, skipped = 0, totallimit = RCUNBOUND;
     bool firstget = true;
     ThorDataLinkMetaInfo inputMeta;
-    Owned<IEngineRowStream> originalInputStream;
+
+    rowcount_t lastTotalLimitState = 0, lastSkipCountState = RCMAX;
+
+    MemoryBuffer currentState; // to determinte whether new lookahead reader required.
```

Comment:
Can be deleted - doesn't appear to be used.

Reply:
will remove